src/server/browserType.ts

```diff
@@ -216,7 +212,7 @@ export abstract class BrowserTypeBase implements BrowserType {
       handleSIGTERM,
       handleSIGHUP,
       progress,
-      pipe: !this._webSocketNotPipe,
+      pipe: true,
```

Comment:
I think this was the only usage of pipe: false, so some more code can be cleaned out in processLauncher.ts